abstracts/abstract_en.tex

```diff
@@ -8,7 +8,16 @@
     It has been employed extensively in acclaimed titles such as \textit{Minecraft} and \textit{No Man's Sky} to create virtually infinite worlds that the player is free to interact with.
     On the other hand, a recent game \textit{Hyperbolica} popularized a novel idea of making the virtual worlds even more interesting by setting them in non-Euclidean spaces.
     The objective of this thesis is to create a small video game incorporating both of the aforementioned concepts.
+    This work describes the implementation of a video game that incorporates both of the aforementioned concepts.
 
-    \noindent \textbf{Keywords:} keyword1, keyword2, ...
+    The game transforms Euclidean space into a hyperbolic or spherical space, which allows for the use of established algorithms.
+    This approach simplifies the implementation of the game logic in different geometries.
+    It also allows for changing the curvature of the space at runtime, which is the best way to showcase the differences between the geometries.
```

Comment:
I'd opt for removing the next two sentences: we are "sort of" changing the curvature only in hyperbolic geometry, and that's still only "sort of" because L stays constant, we only move the camera (change the view matrix).
As for the second sentence, I doubt we could have one scene spanning different geometries, or did you mean like teleporting between different geometreis ? If so this needs to be clarified

Reply:
I think we could have a scene with different geometries as long as the transition between them is smooth

Reply:
But teleportation also works